apps/webcert/src/feature/certificate/UnifiedEdit/UeCheckboxDateRangeList/UeCheckboxDateRangeListField.test.tsx

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable react/function-component-definition */
+import { EnhancedStore } from '@reduxjs/toolkit'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import React, { ComponentProps, useState } from 'react'
+import { Provider } from 'react-redux'
+import { fakeCertificateValue } from '../../../../faker'
+import { certificateMiddleware } from '../../../../store/certificate/certificateMiddleware'
+import { configureApplicationStore } from '../../../../store/configureApplicationStore'
+import { ValueDateRange } from '../../../../types'
+import { formatDateToString } from '../../../../utils'
+import { UeCheckboxDateRangeListField } from './UeCheckboxDateRangeListField'
+
+let testStore: EnhancedStore
+
+const DateRangePickerWrapper: React.FC<Omit<ComponentProps<typeof UeCheckboxDateRangeListField>, 'onChange'>> = ({ value, ...props }) => {
+  const [val, setValue] = useState<ValueDateRange>(value)
+
+  return <UeCheckboxDateRangeListField onChange={setValue} value={val} {...props} />
+}
+
+const renderDefaultComponent = (fromDate = undefined, toDate = undefined, baseWorkHours = '0') => {
```

Comment:
Tests for validation errors are missing

Reply:
tests for validation errors can be found in the parent component `UeCheckboxDateRangeList.test`